resources/function-controller/README.md

```diff
@@ -0,0 +1,60 @@
+# function-controller
```

Comment:
```suggestion
# Function Controller
```
We can use the regular name here, I checked the docs for other resources and they do it the same way.